src/openrct2/scripting/ScRide.hpp

```diff
@@ -229,6 +230,35 @@ namespace OpenRCT2::Scripting
             return "";
         }
 
+        std::string category_get() const
+        {
+            auto ride = GetRide();
+            if (ride != nullptr)
+            {
+                for (auto rideType : ride->GetRideEntry()->ride_type)
```

Comment:
I don't quite get this for, given that it seems like it will always return on the first iteration

Reply:
It is incorrect anyway. You should read the ride type from the ride itself (instead of the ride entry), then use `gRideCategories` to look up the category.